tests/test_utils.py

```diff
@@ -54,6 +56,13 @@ def test_write_labels(tmpdir):
     assert(os.path.exists(file_path))
 
 
+def test_get_ImageDataGenerator(tmpdir):
+    all_images = utils.get_all_images()
+    image = random.sample(all_images, 1)
```

Comment:
It isn't great to use `random` in tests. Just use the first image here.

Reply:
And rename the variable. It's an array of images, so calling it `image` can be confusing.